### ISSUE TITLE
user@example.com stuck in "from email" setting

### ISSUE THREAD

**Issue by summersab** [mautic/mautic#8991]
[//]: # ( Invisible comment: 
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Before you create the issue:
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Search for similar report among other reported issues.
Learn how to troubleshoot at https://www.mautic.org/docs/en/tips/troubleshooting.html
Use drag&drop to attach images or other files )

## Bug Description
After changing Settings > Configuration > Email Settings > E-mail address to send mail from, trying to send a test email results in an error. With AWS SES, the error shown below is returned. With Gmail, the message does indeed send, but it also sends to user@example.com. Somewhere in the code, user@example.com isn't being removed as a placeholder, and the test emails are being sent to both user@example.com and to the "from" address.

| Q   | A
| --- | ---
| Mautic version | 3.0.1
| PHP version | 7.2.31
| Browser | Chromium

### Log errors
Expected response code 250 but got code "554", with message "554 Message rejected: Email address is not verified. The following identities failed the check in region US-EAST-1: user@example.com " (send); user@example.com {"exception":"[object] (Swift_TransportException(code: 554): Expected response code 250 but got code \"554\", with message \"554 Message rejected: Email address is not verified. The following identities failed the check in region US-EAST-1: user@example.com\r\n\" at /opt/bitnami/apps/mautic/htdocs/vendor/swiftmailer/swiftmailer/lib/classes/Swift/Transport/AbstractSmtpTransport.php:457)"} []

[//]: # ( Invisible comment:
Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable. )

**Comment by RCheesley** [comment]
Hi there, have you tried to clear your cache (in var/cache)?

**Comment by RCheesley** [comment]
Closing due to lack of response.

**Comment by dmost714** [comment]
TLDR; Update the email address of the currently logged in user.

I just encountered this issue. I believe the "Send test email" button sends the test email TO the currently logged in user. If you're configuring a new setup, you're logged in as "user" with the placeholder email of "user@example.com".

In AWS SES, make sure the FROM address, and also TO address if you're still in sandbox mode, are verified.

--Dave

**Comment by GrooveDevops** [comment]
is anyone fine solution for this issue
